router/core/transport.go

```diff
@@ -15,6 +21,12 @@ import (
 	"go.uber.org/zap"
 )
 
+const (
+	// dockerInternalHost is the hostnamed used by docker to access the host machine.
+	// We use it for automatic fallbacks when requests to localhost fail
+	dockerInternalHost = "host.docker.internal"
```

Comment:
This won't work on linux. I tested it yesterday after I had to connect docker prometheus with my local router. The solution is https://github.com/wundergraph/cosmo/blob/main/docker-compose.yml#L75 Please add this flag to the router run command in studio.

Reply:
AFAIK `host.docker.internal` only works when the user uses the "bridge" network.